terms.html

```diff
@@ -0,0 +1,86 @@
+
+<p>These rules are designed to maintain a friendly and open atmosphere, and to prevent harassment and discrimination. As such, they are a set of guidelines, but by necessity incomplete. Users violating the spirit of these rules will be treated no differently than users violating a specific rule.</p>
+
+<p>Please note that our rules contain a section on best practices, and users who repeatedly and despite warnings disregard these best practices may be seen to be in violation of our rules.</p>
+
+<p>The moderators may remove accounts who spam the instance, or are suspected of camping just to reserve an account name. Violation of the policies and rules may also lead to account removal at the discretion of the moderators.</p>
+
+<h3>Data Access</h3>
+
+<p>Content on this instance should not be used for the purposes of machine learning or other "research" purposes without the explicit consent of the users involved.</p>
+
+<p>Content on this instance beyond this page should not be archived or indexed wholesale by automated means by any user or service. Active users may export their following lists and post content through the web interface.</p>
+
+
+<hr class="spacer" />
+<hr class="spacer" />
+
+<h2 id="privacy">chaos.social Privacy Policy</h2>
+
+<h3 id="collect">Information collection</h3>
+
+<ul>
+  <li><em>Mandatory account information</em>: Username (always public), e-mail address, and password.</li>
+  <li><em>Optional account information</em>: Display name, biography, profile information fields, profile picture, and header image. Display name, biography, profile picture and header image will always be public.</li>
+  <li><em>Statuses and interactions</em>: We retain all your posts including attachments, and other interactions (such as favourites, follows and reblogs). In addition to the content and people involved, we also store the timestamps for all of the listed data entries. If these interactions impact another server (eg. following, boosting, or messaging a user on a different server), this other server will receive all required information. Public, unlisted, and pinned posts are available publicly. Follower-only posts are available to your followers, and direct messages are available to you and all people mentioned in the message. Please note that since we cannot control other servers, this means that we cannot guarantee the privacy status of your messages as soon as they leave our server.</li>
+  <li><em>Cookies</em>: We use cookies to keep you logged in and save your preferences for future visits.</li>
+  <li><em>Other metadata</em>: We do not log or store your IP address. We retain the name of your browser application to allow you to review your currently logged in sessions for security reasons.</li>
```

Comment:
We should add an exception for error debugging, for a limited time. It's nearly impossible to investigate some problems without logging IPs. For example we had a client a few months ago that rendered the instance unavailable. This wouldn't be solvable without logging.

Reply:
Very good point, I added a note that occasional exceptions can occur for debugging, but that we will delete the IP addresses afterwards.